code/modules/mob/living/carbon/human/human_attackhand.dm

```diff
@@ -218,19 +218,27 @@
 				status += "bruised"
 			if(20 to 40)
 				status += "battered"
-			if(40 to INFINITY)
+			if(40 to 60)
 				status += "mangled"
+			if(60 to 100)
+				status += "brutalized"
+			if(100 to INFINITY)
+				status += "eviscerated"
```

Comment:
Should be mutilated instead of eviscerated since eviscerated is for organ protruding out of the gastrointestinal tract.

![image](https://user-images.githubusercontent.com/6610922/131262361-047050d6-ec1a-471f-90d4-0270efa8460c.png)

Reply:
ok